.github/workflows/dockerhub.yml

```diff
@@ -26,16 +30,20 @@ jobs:
     runs-on: ubuntu-latest
     env:
       name: ${{ inputs.Application }}
+    outputs:
+      docker_tag: ${{ steps.build.outputs.docker_tag }}
     steps:
       - uses: actions/checkout@v3
       - name: build
+        id: build
```

Comment:
I think the step ID is needed so we can reference the step in the outputs above (`${{ steps.build.outputs.docker_tag }}`)